docs/source/pillars/computing_technology.md

```diff
@@ -104,59 +132,90 @@ _The ability to run analyses requiring more compute resources, or more specialis
+* - You must have a documented procedure for deploying infrastructure.
+  - This might, for instance, be a handbook that is followed or a set of automated scripts.
   - Mandatory
-* - Your TRE should record which datasets are accessed, when and by who.
-  - This helps maintain auditability of how sensitive data has been used.
+* - You should, where possible, automate any repeatable aspects of your deployment.
+  - This might involve using infrastructure-as-code tools or simply a series of scripts.
   - Recommended
-* - Your TRE should record computational resource usage at the user or aggregate level.
-  - This is useful for optimising allocation of resources, and managing costs.
+* - You must test changes before they are used in production.
+  - This might involve a separate development environment or another system for testing.
+  - Mandatory
+* - You should have separate technical environments for development and testing infrastructure changes before they are committed to production.
+  - If possible, you should automate application of changes between development and production environments.
+    Consider the costs and practicality of whether this will work for your situation.
   - Recommended
 ```
 
-## Network management
+### Infrastructure decommissioning
 
-_The ability of the TRE operator to administer and secure network infrastructure using applications, tools and processes._
+The process of retiring or removing infrastructure assets that are no longer needed or outdated, ensuring proper data handling and disposal.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-end-user-network-management
+:name: tab-infrastructure-decommissioning
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a documented procedure for removing infrastructure when it is no longer needed
+  - Removing unused infrastructure not only reduces costs and management burden but also reduces the attack surface of a TRE and reduces the risk of unaddressed vulnerabilities.
+  - Mandatory
+```
+
+### Availability management
+
+The process of ensuring all IT infrastructure meets the agreed levels of availability.
+
+```{list-table}
+:header-rows: 1
+:name: tab-infrastructure-availability
+
+* - Statement
+  - Guidance
+  - Importance
+* - You should understand the availability and uptime guarantees of any providers that you rely on.
+  - For remote TREs this might include your cloud provider(s) and/or data centre operators.
+    For on-premises TREs, it might be worth considering your ISP and electricity provider.
+  - Recommended
+* - You should develop an availability target or statement and share this with your users.
+  - Understanding how and when the TRE might be unavailable will help your projects in planning their work.
+  - Recommended
+```
+
+### Network management
+
+An application used to manage and control network devices, ensuring proper functioning, security, and performance of the network.
```

Comment:
```suggestion
Managing network infrastructure, ensuring proper functioning, security, and performance.
```

Reply:
I think @machintim wants to make it clear which things are processes/applications/data assets. Can you reword in a way that isn't ambiguous?

Reply:
```suggestion
An application used to manage network infrastructure, ensuring proper functioning, security, and performance.
```